nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/with-blank-cells.csv

```diff
@@ -0,0 +1,8 @@
+A,B,C,D
```

Comment:
@m-hogue Thanks for your comment! If a row only contains empty cells, then it will be ignored. Since we're processing an Excel sheet as an XML document, such row wouldn't be passed to our code. Excel XML only contains rows and cells those have values.